quickbooks/objects/vendorcredit.py

```diff
@@ -44,6 +44,7 @@ def __init__(self):
         self.FromAccountRef = None
         self.ToAccountRef = None
 
+        self.LinkedTxn = []
```

Comment:
nothing wrong with this, but it seemingly doesn't do anything?

Reply:
Its populated dynamically by a different part of the code